Master

eliso7 opened the pull request.

taw commented:
@axxroytovu Does it look right to you?

axxroytovu commented on data/boosters/fdn-prerelease.yaml:
```diff
@@ -0,0 +1,7 @@
+  prerelease_promo: 1
+sheets:
+  prerelease_promo:
+    filter: 'e:p{set} promo:prerelease number<=257s'
```
Is there a particular reason you included the number range filter? It's not _wrong_ per-se, but it doesn't really do anything since searching for `number>=258` returns no results.

eliso7 replied:
The query for either appears to return the same results without any constraint on number.

eliso7 commented:
I have added the other missing prerelease packs and removed the uneccesary number contraint in the query for lci and modified the query to omit adventure card faces.

axxroytovu commented:
Looks good. 👍 

Doesn't need to change for this PR, but in the future you don't need to filter out adventure/split/backside cards. The script automatically makes sure to only grab the front face of a card.
